lib/helpers.ts

```diff
@@ -51,6 +53,25 @@ export function extractFormData(submission: Submission): Array<string> {
   return dataCollector;
 }
 
+// Sends an email using the Notify API
```

Comment:
@timarney I was thinking about how to unit test this function. I'm not really sure about Notify dependency and what to mock. I guess the test could throw similar errors as Notify but if Notify changes..
Do you have any ideas? 
If this isn't trivial, the sendEmail util function could be done in another task

Reply:
Actually I should probably ask questions over slack - never mind :)